Sources/Prometheus/MetricTypes/PromMetric.swift

```diff
@@ -1,3 +1,5 @@
+import NIO
```

Comment:
side topic question: don't all sources have to start with a copyright?

Reply:
Euhm, I honestly don't know. I don't think they HAVE to, but I'll double check.